src/common/src/array/vis.rs

```diff
@@ -68,6 +68,14 @@ impl Vis {
             Vis::Compact(c) => VisRef::Compact(*c),
         }
     }
+
+    /// Consumes this `Vis` and returns the inner `Bitmap` if not compact.
+    pub fn into_bitmap(self) -> Option<Bitmap> {
```

Comment:
```suggestion
    pub fn into_vis(self) -> Option<Bitmap> {
```

Reply:
Seems we're using the term `vis` for the `Vis2` while `visibility` for the old bitmap representation. I'd like to name it `into_visibility`. 🤣